### PR TITLE
[release-9.1] Fix and improve popovers in channels.

### PR DESCRIPTION
**Pull request by EreMaijala**
- Switches to manual focus to ensure that the popover stays visible as long as needed.
- Fixes popover record data table visibility.

### CODE REVIEW

**Review by demiankatz — APPROVED** [review]
This looks good to me, @EreMaijala, and all tests are passing.

I'll just wait until @sturkel89 has a chance to do some testing as well tomorrow before merging, in case she finds any anomalies that I have missed.

**Comment by demiankatz** [comment]
@sturkel89, the main purpose of this PR is to resolve some remaining anomalies related to the behavior of the channel popover that appears when you click on a channel item. It's probably worth double-checking that popovers still display correctly in all three themes (though I have no reason to believe that they won't) and to be sure that other channel behavior (loading extra channels, scrolling, etc.) still works. I have tried several things myself without finding any issues, but another pair of eyes doesn't hurt!

**Review by sturkel89 — APPROVED** [review]
I tested this branch in Chrome and Firefox, looking at all three themes. I can't get it to behave badly! All the popovers displayed correctly, even if I looked at the same record in different channels repeatedly.

I think this code is good!

**Pull request merged**
